docs/src/content/docs/de/resources/plugins.mdx

```diff
@@ -184,4 +184,9 @@ Diese Community-Tools und Integrationen können verwendet werden, um deiner Star
 		title="@hideoo/starlight-plugin"
 		description="Ein Generator, um schnell Starlight-Plugins zu erstellen."
 	/>
+	<LinkCard
+		href="https://github.com/Linkerin/starlight-to-pdf"
+		title="starlight-to-pdf"
+		description="Ein CLI-Werkzeug zur Konvertierung von Starlight-Websites in PDF-Dateien."
```

Comment:
In the Astro docs glossary for German (https://github.com/withastro/docs/blob/main/i18n-guides/deutsch.md), “CLI” is translated as “Kommandozeile”, so I guess the consistent translation here would be:

```suggestion
		description="Ein Kommandozeilenwerkzeug zur Konvertierung von Starlight-Websites in PDF-Dateien."
```

What do you think?